### PR TITLE
added page for #103

### LINKED ISSUE #103: Ouput Redirection Operator >
>

### PR DESCRIPTION
**Pull request by Lilisys**
# Advanced HW 9 submission

Closes #103 

Name: Jessica Wu
Uniqname: jesswu

### CODE REVIEW

File: _commands/scripting/basicStdOutRedirect.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+#### Combining with `diff`
+~~~ bash
+$ ./p3 > outfile.txt | diff outfile.txt correct.txt
+1c1
+< Hello World!
+---
+> Hello World!!
+~~~
+
```
Comment (the-alex) [inline _commands/scripting/basicStdOutRedirect.md]: line 21: I don't think this line produces that output. This is the exact gotcha we put on a homework earlier in the semester. The `>` opens a new file with the name `outfile.txt` and erases the contents and THEN diffs the two files. 

Here's an example of my terminal session.

~~~bash
λ: ./main
hello world
λ: cat outfile.correct
hello world!!!
λ: ./main > outfile.out && diff outfile.out outfile.correct
1c1
< hello world
---
> hello world!!!
λ: ./main > outfile.out | diff outfile.out outfile.correct
0a1
> hello world!!!
~~~

If you want to run two commands on the same line, you can use the `&&` or `;` between commands. The only difference is that for the command after `&&` to run, the first one needs to return successfully. `;` is just like ending the line early and starting another.

~~~bash
λ: false && echo "hello"  # fails and prints nothing. AKA a 'short circuit'
λ: false; echo "hello"
hello
~~~

**Comment by Lilisys** [comment]
@the-alex Thanks for pointing this out! I'll fix it now.

File: _commands/scripting/basicStdOutRedirect.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ 
+
+#### Splitting stdout and stderr
+~~~ bash
+$ ./a.out
+Hello from cerr!
+Hello World!
+$ ./a.out > std
+Hello from cerr!
+~~~
```
Comment (the-alex) [inline _commands/scripting/basicStdOutRedirect.md]: Could you add another "break it down" section for this last example?

**Comment by Lilisys** [comment]
@the-alex dangit, I remember writing that section yesterday but i guess i accidentally deleted it. It's fixed now.

**Pull request merged**